typescript/create-onchain-agent/templates/actionProvider/actionProvider.njk

```diff
@@ -0,0 +1,20 @@
+import { customActionProvider{% if walletProvider !== "none" %}, {{ walletProvider }}{% endif %} } from "@coinbase/agentkit";
+import { {{ schemaName }} } from "./schemas";
+import { z } from "zod";
+
+/**
+ * Creates a {{ className }} action provider.
+ */
```

Comment:
Let's add a callout that, if the dev wants to add additional actions to the action provider, they can change the input to `customActionProvider` to be an array, like so:
```ts
export const myactionActionProvider = () => customActionProvider<EvmWalletProvider>([
  {
    name: "myaction_action",
    description: `This tool will perform a MyactionActionProvider operation.`,
    schema: MyactionActionSchema,
    invoke: async (wallet: EvmWalletProvider, args: z.infer<typeof MyactionActionSchema>) => {
      try {
        // Do work here
        return `Successfully performed myaction_action and returned the response`;
      } catch (error) {
        return `Error performing myaction_action: Error: ${error}`;
      }
    },
  },
]);
```